server.js

```diff
@@ -10,7 +11,7 @@ app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 // Serve up static assets (usually on heroku)
 if (process.env.NODE_ENV === "production") {
-  app.use(express.static("client/build"));
+  app.use(express.static("client/public"));
```

Comment:
not sure if this will break heroku... if it doesn't work after merge, try setting this back to build and see what happens

Reply:
I think we need the public - the build was put in when I ran the npm run build package